doc/packages/uv.section.md

```diff
@@ -0,0 +1,19 @@
+# uv {#sec-uv}
+
+`uv` is an extremely fast Python package installer and resolver, written in Rust.
+It manages project dependencies and environments, with support for lockfiles, workspaces, and more.
+
+Due to `uv` being unaware that it is running on a NixOS system, by default, it will fetch dynamically-linked Python executables that will fail to run, as NixOS cannot run executables intended for generic Linux environments out of the box.
```

Comment:
How about we link to https://nix.dev/guides/faq.html#how-to-run-non-nix-executables?